doc/DeveloperGuide.md

```diff
@@ -87,6 +87,23 @@ Use case ends.
 
 > 3a1. AddressBook shows an error message <br>
   Use case resumes at step 2
+  
+#### Use case: Add person
+
+**MSS**
+
+1. User requests to add a specific person in the list
+2. AddressBook add the person <br>
+3. AddressBook shows the person added
```

Comment:
Use cases should depict the back and forth interaction between the user and the system. For every step the user does, the system should do something in return (and vice-versa).